.gitignore

```diff
@@ -1,6 +1,10 @@
 node_modules/
 tmp/
+build/
```

Comment:
qmlweb2 creates it ;)
I think .gitignore is a place to ask "why not?", rather than "why?", because what ever helps one developer should go in ;)
But feel free to remove, if you don't agree :)